src/index.jsx

```diff
@@ -1,83 +1,80 @@
 import TitleBar from "./components/TitleBar.jsx"
+import RatingsReviews from './components/RatingsReviews/RatingsReviews.jsx';
 const root = createRoot(document.getElementById("root"));
 
 
-
-
 class App extends React.Component {
   constructor(props) {
     super (props)
     this.state = {
-      products: [
-      ],
-
-      product: {
-      },
+      products: [],
+      product: {},
+      meta: {}
     }
   }
 
+
   selectProduct (product) {
     this.setState({ 'product': product })
-    //TODO: implement product selection function
   }
 
   componentDidMount() {
-    //GET request for a list of products and their IDs
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/products/',
-      {headers: {'Authorization': `${API_KEY}`},
-      params: {count: 5, page: 1}})
-        .then((res) => {
-        this.setState({ 'products': res.data, 'product': res.data[0] })})
-        .catch((err) => console.log(err));
-    //GET request for Q&A at a product_id
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/qa/questions',
-      {headers: {'Authorization': `${API_KEY}`},
-      params: {count: 5, page: 1, product_id: 66673 }})
-        .then((res) => console.log('Q&A at product_id: ', res.data))
-        .catch((err) => console.log(err));
-    //GET request for reviews at product ID
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/reviews',
-      {headers: {'Authorization': `${API_KEY}`},
-      params: {count: 5, page: 1, product_id: 66673, sort: 'newest' }})
-        .then((res) => console.log('Reviews at product_id: ', res.data))
-        .catch((err) => console.log(err));
-    //GET request for Cart (Currently empty but can be filled with POST request)
-    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/cart',
-      {headers: {'Authorization': `${API_KEY}`}})
-        .then((res) => console.log('Cart Data: ', res.data))
-        .catch((err) => console.log(err));
-      //Example POST request for adding interactions to the DB
-      // axios.post('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/interactions',{element: 'Selector for the clicked element', widget: 'Name of widget in which click occured', time: 'Time the click occured'},
-      // {headers: {'Authorization': `${API_KEY}`}})
-      //   .then((res) => console.log(res.data))
-      //   .catch((err) => console.log(err));
-
+    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/products/',{
+      headers: {'Authorization': `${API_KEY}`},
+      params: {
+        count: 5,
+        page: 1
+      }})
+    .then(res => {
+      this.setState({ products: res.data })
+      console.log('Array of Products: ', res.data)
+      return axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/reviews/meta', {
+        headers: {'Authorization': `${API_KEY}`},
+        params: {product_id: 66642}
+      })
+    })
+    .then(res => {
+      this.setState({ meta: res.data })
```

Comment:
We could maybe call this reviewMeta since (I'm not sure but) I think Zach will have a metadata call also.

Reply:
Agreed, updated!